fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -1069,6 +1069,97 @@ def assert_state(self, state: TrainingState) -> None:
             self.training_state == state
         ), f"expected to be in state {state} but current state is {self.training_state}"
 
+    @staticmethod
+    @contextlib.contextmanager
+    def enable_wrap(**kwargs) -> Iterator[None]:
+        """
+        Context manager to wrap modules in FullyShardedDataParallel.
+        Useful for when you'd like to apply the same parameters to all child modules that you wrap.
```

Comment:
Let's add more context about the use cases, how about:
```suggestion
        Context manager to wrap modules in FullyShardedDataParallel.
        
        Useful for when you'd like to apply the same parameters to all child modules
        that you wrap. A particularly important use case is wrapping large layers so
        that they get sharded (in-place) during initialization, to avoid running out of
        system memory.
```